frontend/src/Editor/WidgetManager/widgetConfig.js

```diff
@@ -821,15 +821,64 @@ export const widgets = [
       showOnMobile: { type: 'toggle', displayName: 'Show on mobile' },
     },
     properties: {
-      defaultStartDate: { type: 'code', displayName: 'Default start date' },
-      defaultEndDate: { type: 'code', displayName: 'Default end date' },
-      format: { type: 'code', displayName: 'Format' },
+      defaultStartDate: {
+        type: 'code',
+        displayName: 'Default start date',
+        validation: {
+          schema: {
+            type: 'string',
+          },
+        },
+      },
+      defaultEndDate: {
+        type: 'code',
+        displayName: 'Default end date',
+        validation: {
+          schema: {
+            type: 'string',
+          },
+        },
+      },
+      format: {
+        type: 'code',
+        displayName: 'Format',
+        validation: {
+          schema: {
+            type: 'string',
```

Comment:
@manishkushare Validating `Format` field is not working. Can you check it?

Reply:
@kavinvenkatachalam, Fixed the issue.